protos/flyteidl/admin/schedule.proto

```diff
@@ -16,6 +16,13 @@ message FixedRate {
     FixedRateUnit unit = 2;
 }
 
+// Styx style schedule
+// More details can be found at https://github.com/spotify/styx#schedule-string
+message StyxSchedule {
```

Comment:
Should we call it StyxSchedule or just `CronScheduleWithOffset`
We should also add a note here
```python
Supported only by some schedulers for example Spotify/StyxSchedule. Your platform may not have this enabled.
And correspondingly we should fail registration if using a schedule that is not supported
```

Reply:
Totally. This would look a lot better. I will fix that.